test/testUtils/imageUtils.py

```diff
@@ -15,8 +15,6 @@
 #
```

Comment:
Removed a bunch of settings changes that were happening under the hood and moved them to an optional step when opening scenes (except `camera` as that one was never used). I kept `defaultRenderGlobals.imageFormat` because this is set based on the extension of the file name being passed in.